Dockerfile

```diff
@@ -1,2 +1,23 @@
-FROM node:0.10-onbuild
-EXPOSE 3000
\ No newline at end of file
+FROM node:0.10
+
+RUN mkdir -p /opt/wftda/bouttime
+WORKDIR /opt/wftda/bouttime
+
+ARG NODE_ENV
```

Comment:
This came from the onbuild (no excuse!). I had thought there might be a need for a production build for distribution, but given this docker image is development focused it's unlikely. I'll take it out.